### PR TITLE
OSSMDOC-354: Fix incorrect link in Release Notes.

### PR DESCRIPTION
**Pull request by JStickler**
Fixes link to incorrect JIRA in Known Issues section.

### CODE REVIEW

**Review by bergerhoffer — COMMENTED** [review]
As long as you also meant to delete the bullet above (in addition to changing the Jira link), then this LGTM

File: modules/ossm-rn-known-issues.adoc [hunk 1]
```diff
@@ -51,6 +49,8 @@ spec:
           - 15020
 ----
 +
+* link:https://issues.jboss.org/browse/MAISTRA-1947[MAISTRA-1947] _Technology Preview_  Updates to ServiceMeshExtensions are not applied.  The workaround is to remove and recreate the ServiceMeshExtensions.
```
Comment (rolfedh) [inline modules/ossm-rn-known-issues.adoc:52]:
```suggestion
* link:https://issues.jboss.org/browse/MAISTRA-1947[MAISTRA-1947] _Technology Preview_  Updates to `ServiceMeshExtensions` are not applied.  

Workaround: Remove and recreate the `ServiceMeshExtensions`.
```

File: modules/ossm-rn-known-issues.adoc [hunk 2]
```diff
@@ -34,9 +34,7 @@ These are the known issues in {ProductName}:
 
 * link:https://issues.redhat.com/browse/OSSM-285[OSSM-285] When trying to access the Kiali console, receive the following error message "Error trying to get OAuth Metadata". The workaround is to restart the Kiali pod.
 
-* link:https://issues.jboss.org/browse/MAISTRA-1947[MAISTRA-1947] _Technology Preview_  Updates to ServiceMeshExtensions are not applied.  The workaround is to remove and recreate the ServiceMeshExtensions.
-
-* link:https://issues.jboss.org/browse/MAISTRA-1621[MAISTRA-1621]  _Migration to 2.0_  Prometheus scraping (`spec.addons.prometheus.scrape` set to `true`) does not work when mTLS is enabled.  Additionally, Kiali displays extraneous graph data when mTLS is disabled.
+* link:https://issues.jboss.org/browse/MAISTRA-1959[MAISTRA-1959]  _Migration to 2.0_  Prometheus scraping (`spec.addons.prometheus.scrape` set to `true`) does not work when mTLS is enabled.  Additionally, Kiali displays extraneous graph data when mTLS is disabled.
 +
 This problem can be addressed by excluding port 15020 from proxy configuration, for example,
```
Comment (rolfedh) [inline modules/ossm-rn-known-issues.adoc:39]:
```suggestion
Workaround: You can exclude port 15020 from proxy configuration. For example:
```

Comment (rolfedh) [reply modules/ossm-rn-known-issues.adoc:39]: This suggestion is based on the recently-added "Release Notes" section in the Supplemental Style Guide: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes

**Review by rolfedh — APPROVED** [review]

**Pull request merged**